gcloud-java-examples/README.md

```diff
@@ -89,6 +89,28 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`, illustrating the use of the gcloud-java-nio jar.
+
+     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
```

Comment:
Nit: do you mind adding "JSON" between "Storage" and "API"?  I believe that's the name of the API verbatim, and there are two that come up that are similar if you search "storage" on the developers console.  (I realize you copied what we had for StorageExample, so not your bug.  Related to that, do you mind adding "JSON" up in StorageExample's docs too?)